cmd/genconversion/conversion.go

```diff
@@ -71,12 +72,19 @@ func main() {
 	}
 
 	versionPath := path.Join(pkgBase, group, version)
+	if group == "experimental" {
+		//TODO: we should rename the direcotry /expapi to /experimental, so directory path match groupVerson
+		versionPath = path.Join(pkgBase, "expapi", version)
+	}
 	generator := pkg_runtime.NewConversionGenerator(api.Scheme.Raw(), versionPath)
 	apiShort := generator.AddImport(path.Join(pkgBase, "api"))
 	generator.AddImport(path.Join(pkgBase, "api/resource"))
 	// TODO(wojtek-t): Change the overwrites to a flag.
 	generator.OverwritePackage(version, "")
-	for _, knownType := range api.Scheme.KnownTypes(version) {
+	if *groupVersion == "api/v1" {
```

Comment:
Is this to not break existing clients who might be passing api/v1 explicitly?
If yes, then shouldnt we be doing this at the top? Maybe just after flag.Parse.
Also add a comment saying this to support old clients, who might still be sending api/v1, and we can delete it once we stop supporting v1